.github/workflows/stig.yaml

```
@@ -20,7 +20,7 @@ jobs:
       - name: Build the image
         run: ./build ${{ matrix.platform}}-stig
       - name: Upload build logs
-        uses: actions/upload-artifact@v4
+      - uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # pin@v4.4.0
```

```suggestion
        uses: actions/upload-artifact@50769540e7f4bd5e21e526ee35c689e35e0d6874 # pin@v4.4.0
```

wait a second, this introduces a syntax error

---

fixed